otherlibs/systhreads/st_stubs.c

```diff
@@ -364,6 +364,10 @@ static void caml_thread_remove_and_free(caml_thread_t th)
 
 static void caml_thread_reinitialize(void)
 {
+  /* Note: "If a multi-threaded process calls fork() [...] the child
```

Comment:
These are useful comments. This commit LGTM.

Reply:
Could the comment be a little more specific about the actual precondition?  `caml_thread_reinitialize` is called as part of the `caml_atfork_hook`, and therefore this function is executed right after a `fork` (and then the rest of the proposed change makes more sense in context).

Reply:
Thanks, I took your phrasing verbatim.